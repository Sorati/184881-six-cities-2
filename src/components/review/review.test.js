import renderer from 'react-test-renderer';
import Review from '../review/review';

it(`Review correctly renders with data`, () => {

  const review = {
    user: {
      photo: `/img/apartment-02.jpg`,
      name: `Max`
    },
    rating: 72,
    text: `Great place!`,
    date: `2019-07-12`
  };

  const {user, rating, text, date} = review;

  const tree = renderer
    .create(<Review user={user} rating={rating} text={text} date={date}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
