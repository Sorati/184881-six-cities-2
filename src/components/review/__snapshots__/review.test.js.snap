// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review correctly renders with data 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height={54}
        src="/img/apartment-02.jpg"
        width={54}
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Max
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "72%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Great place!
    </p>
    <time
      className="reviews__time"
      dateTime="2019-07-12"
    >
      July 2019
    </time>
  </div>
</li>
`;
